.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: 'bug:'
+labels: bug
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Run cmd '...'
+2. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Please complete the following information):**
+ - OS: [e.g. macOS Big Sur 11.5.2 ]
+ - Version [e.g. v0.15.0]
```

Comment:
Should we be explicit about this version, I think it is the version of our Terraform provider, right?
```suggestion
 - Provider Version [e.g. v0.15.0]
```